### PR TITLE
Notify user when a command execution error occurs

### PR DESCRIPTION
**Pull request by Octol1ttle**
With this PR, whenever command execution fails, the user will get an error message with details of the error that can be passed on to developers

An unrelated minor change: errors caused by task cancellations will no longer be logged

### CODE REVIEW

File: locale/Messages.tt-ru.resx [hunk 1]
```diff
@@ -570,4 +570,7 @@
   <data name="MessagesClearedFiltered" xml:space="preserve">
       <value>вырезано {0} забавных сообщений от {1}</value>
   </data>
+  <data name="CommandExecutionFailed" xml:space="preserve">
+      <value>произошёл пиздец в команде, удачи чел.</value>
```
Comment (mctaylors) [inline locale/Messages.tt-ru.resx:574]: not that rude :(
```suggestion
      <value>произошёл тотальный разнос в команде, удачи.</value>
```

**Review by mctaylors — CHANGES_REQUESTED** [review]

**Comment by mctaylors** [comment]
Do you mind if I add a line about "contacting the system administrator if the error occurs again" to the footer?

**Comment by Octol1ttle** [comment]
> Do you mind if I add a line about "contacting the system administrator if the error occurs again" to the footer?

"system administrator" is too vague IMO

**Comment by mctaylors** [comment]
> > Do you mind if I add a line about "contacting the system administrator if the error occurs again" to the footer?
> 
> "system administrator" is too vague IMO

"developers"?

**Comment by Octol1ttle** [comment]
> > > Do you mind if I add a line about "contacting the system administrator if the error occurs again" to the footer?
> > 
> > 
> > "system administrator" is too vague IMO
> 
> "developers"?

sure

File: locale/Messages.ru.resx [hunk 2]
```diff
@@ -570,4 +570,10 @@
   <data name="MessagesClearedFiltered" xml:space="preserve">
       <value>Очищено {0} сообщений от {1}</value>
   </data>
+  <data name="CommandExecutionFailed" xml:space="preserve">
+      <value>Произошла ошибка при выполнении команды, повтори попытку позже.</value>
+  </data>
+  <data name="ContactDevelopers" xml:space="preserve">
+      <value>Обратитесь к разработчикам, если проблема возникнет снова.</value>
```
Comment (Octol1ttle) [inline locale/Messages.ru.resx:577]: let's try to обращаться на "ты"

**Review by mctaylors — APPROVED** [review]

**Review by mctaylors — APPROVED** [review]

**Pull request merged**